src/renderer/components/wallet/txs/SendFormBTC.txt

```diff
@@ -0,0 +1,198 @@
+import { SendTxParams, AddressValidation } from '../../../services/bitcoin/types'
+import { AssetsWithBalance, AssetWithBalance } from '../../../services/wallet/types'
+import { Input, InputBigNumber } from '../../uielements/input'
+import AccountSelector from '../AccountSelector'
+import * as Styled from './Form.style'
+import { validateTxAmountInput } from './util'
+
+type FormValues = {
+  recipient: string
+  amount: string
+}
+
+type Props = {
+  assetsWB: AssetsWithBalance
+  assetWB: AssetWithBalance
+  onSubmit: ({ to, amount, memo }: SendTxParams) => void
+  changeAssetHandler: (asset: Asset) => void
+  isLoading: boolean
+  addressValidation: AddressValidation
+  feeOptions: O.Option<FeeOptions>
+}
+
+export const SendForm: React.FC<Props> = (props): JSX.Element => {
+  const {
+    onSubmit: onSubmitProp,
+    changeAssetHandler,
+    assetsWB,
+    assetWB,
+    isLoading = false,
+    addressValidation,
+    feeOptions: oFeeOptions
+  } = props
+  const intl = useIntl()
+
+  const [form] = Form.useForm<FormValues>()
+
+  const feeLabel = useMemo(
+    () =>
+      FP.pipe(
+        oFeeOptions,
+        O.fold(
+          () => '--',
+          // TODO (@Veado) Which
+          ({ regular }) => `${AssetSymbol.BTC} ${trimZeros(regular.feeRate.toString())} `
+        )
+      ),
+    [oFeeOptions]
+  )
+
+  const isFeeError = useMemo(() => {
+    return FP.pipe(
+      oFeeOptions,
+      O.fold(
+        // Missing (or loading) fees does not mean we can't sent something. No error then.
+        () => !O.isNone(oFee),
+        ([fee, bnbAmount]) => bnbAmount.amount().isLessThan(fee.amount())
+      )
+    )
+  }, [oFeeOptions])
+
+  const renderFeeError = useMemo(() => {
+    if (!isFeeError) return <></>
+
+    const amount = FP.pipe(
+      oBnbAmount,
+      // no bnb asset == zero amount
+      O.getOrElse(() => assetAmount(0))
+    )
+
+    const msg = intl.formatMessage(
+      { id: 'wallet.errors.fee.notCovered' },
+      {
+        fee: formatAssetAmount(amount, 6),
+        balance: `${formatAssetAmount(amount, 8)} ${BNB_SYMBOL}`
+      }
+    )
+
+    return (
+      <Styled.StyledLabel size="big" color="error">
+        {msg}
+      </Styled.StyledLabel>
+    )
+  }, [intl, isFeeError])
+
+  const addressValidator = useCallback(
+    async (_: unknown, value: string) => {
+      if (!value) {
+        return Promise.reject(intl.formatMessage({ id: 'wallet.errors.address.empty' }))
+      }
+      if (!addressValidation(value.toLowerCase())) {
+        return Promise.reject(intl.formatMessage({ id: 'wallet.errors.address.invalid' }))
+      }
+    },
+    [addressValidation, intl]
+  )
+
+  const amountValidator = useCallback(
+    async (_: unknown, value: BigNumber) => {
+      // max amount for bnb
+      const maxBnbAmount = FP.pipe(
+        sequenceTOption(oFee, oBnbAmount),
+        O.fold(
+          // Set maxAmount to zero if we dont know anything about bnb and fee amounts
+          () => bn(0),
```

Comment:
and here we could use [ZERO_BN](https://github.com/thorchain/asgardex-electron/blob/master/src/renderer/const.ts#L58) ?